iscsi/initiator.go

```diff
@@ -166,6 +181,34 @@ func IsTargetLoggedIn(ip, target string, ne *util.NamespaceExecutor) bool {
 	return found
 }
 
+func manualScanSession(ip, target string, ne *util.NamespaceExecutor) error {
+	opts := []string{
+		"-m", "node",
+		"-T", target,
+		"-p", ip,
+		"--rescan",
+	}
+	_, err := ne.ExecuteWithTimeout(ScanTimeout, iscsiBinary, opts)
+	return err
+}
+
+func getIscsiNodeSessionScanMode(ip, target string, ne *util.NamespaceExecutor) string {
+	opts := []string{
+		"-m", "node",
+		"-T", target,
+		"-p", ip,
+		"-o", "show",
+	}
+	output, err := ne.ExecuteWithTimeout(ScanTimeout, iscsiBinary, opts)
+	if err != nil {
+		logrus.Warnf("Failed to get node.session.scan mode: %v", err)
```

Comment:
we should return here if it cannot be done.

Reply:
fixed